source/administration/config-settings.rst

```diff
@@ -289,6 +289,20 @@ Enable Team Directory
 | This feature's ``config.json`` setting is ``"EnableTeamListing": false`` with options ``true`` and ``false`` for above settings respectively.                        |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Teammate Name Display
+^^^^^^^^^^^^^^^^^^^^^
+Specifies how names are displayed in the user interface.
+
+**Show username**: Displays the user's username.
+
+**Show nickname if one exists**: Displays the user's nickname. If the user does not have a nickname, their full name is displayed.
+
+**Show first and last name**: Displays the user's full name. Recommended when using SAML or LDAP if first name and last name attributes are configured.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"TeammateNameDisplay": "username"`` with options ``username``, ``nickname_full_name``, and ``full_name``. |
```

Comment:
Yep. More thought needs to go into these kinds of names, because bad choices tend to last forever (for backwards compatibility).